csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -63,6 +63,13 @@
           Pack="true"
           Visible="false"
     />
+
+    <None Include="Microsoft.ML.OnnxRuntime.targets"
+          PackagePath="\build"
```

Comment:
One mistake I've made in the past is that when your NuGet package has just a `lib\netstandardX.Y` library in it, NuGet will ensure that a users project supports `netstandardX.Y` and block adding this package if it doesn't.

However, once you then add a `build\PackageId.props|targets` file, now NuGet thinks your package supports ANY project type. This is usually wrong.

Instead, since your package only supports `netstandard1.1`, this .targets file should go into the `build\netstandard1.1` folder.

See [https://github.com/dotnet/machinelearning/pull/370](https://github.com/dotnet/machinelearning/pull/370) for where I made the same mistake in ML.NET.